migrations/migrations/20181023175544-users-model.js

```diff
@@ -27,6 +27,11 @@ module.exports = {
         type: Sequelize.STRING,
         allowNull: false
       },
+      admin: {
```

Comment:
The idea is to create a new migration with the adittion of the field.

Once a migration is executed, the orm will ignore it, so any change won't be recognize automatically and won't be added. 
By doing it the way you did, you probably had to drop and create the db from scratch (or add the field manually via psql) at this point, and if you were working with some other devs they also had to drop and create the db form scratch and also if the project were to be deployed in some remote environment. Which is what we intent to avoid with the addition of the migrations.
If you have any doubt about this (or any other correction or subject) let me know and we can talk a little bit more.

Reply:
Fixed